### PR TITLE
blog styling change

### PR DESCRIPTION
**Pull request by haseebc**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def markdown(description)
       underline: true,
       highlight: true,
       quote: true,
-      footnotes: true
+      footnotes: true,
```
Comment (franamadis) [inline app/helpers/application_helper.rb]: you dont really need the coma if its the last value, but its a good practice.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ def block_code(code, language)
 
   end
   def markdown(description)
-    renderer = HTMLwithPygments.new(hard_wrap: true, filter_html: true)
```
Comment (franamadis) [inline app/helpers/application_helper.rb]: is this related to the gem pygments.rb???
i got no clue.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (haseebc) [reply app/helpers/application_helper.rb]: yeah I only just noticed that at the end. good spot :-)

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (haseebc) [reply app/helpers/application_helper.rb]: Its for redcarpet (blog markdown gem), the change lets me as the author insert html into the blog page. its secure as we're only going to have us as authors so its cool.

Comment (franamadis) [reply app/helpers/application_helper.rb]: Okay... you sound so convincing. 👍

**Pull request merged**